src/services/goToDefinition.ts

```diff
@@ -113,6 +117,26 @@ namespace ts.GoToDefinition {
         }
     }
 
+    function getDefinitionFromOverriddenMember(typeChecker: TypeChecker, node: Node) {
+        const classElement = findAncestor(node, isClassElement);
+        if (!(classElement && classElement.name)) return;
+
+        const baseDeclaration = findAncestor(classElement, isClassLike);
+        if (!baseDeclaration) return;
+
+        const baseTypeNode = getEffectiveBaseTypeNode(baseDeclaration);
+        const baseType = baseTypeNode ? typeChecker.getTypeAtLocation(baseTypeNode) : undefined;
+        if (!baseType) return;
+
+        const name = unescapeLeadingUnderscores(getTextOfPropertyName(classElement.name));
+        const symbol = hasStaticModifier(classElement)
+            ? typeChecker.getPropertyOfType(typeChecker.getTypeOfSymbolAtLocation(baseType.symbol, baseDeclaration), name)

```

Comment:
Does this work through two levels of inheritance?

```ts
class Bottom {
  /*2*/m() {}
}

class Middle extends Bottom {}

class Top extends Middle {
  /*1*/override m() {}
}
```

Reply:
I've added goToDefinitionOverriddenMember14 to cover that case

Reply:
I've added `goToDefinitionOverriddenMember14`, `goToDefinitionOverriddenMember15` to cover that case